fuzz/src/full_stack.rs

```diff
@@ -624,14 +624,14 @@ mod tests {
 
 		// Writing new code generating transactions and see a new failure ? Don't forget to add input for the FuzzEstimator !
 
-		// 0000000000000000000000000000000000000000000000000000000000000000 - our network key
+		// 0100000000000000000000000000000000000000000000000000000000000000 - our network key
```

Comment:
They extend from a few things (which is why there's so many of them). This particular one extends from the fact that secp256k1's fuzzing mode now requires the private key *not* be all-0s (in fact it enforces the regular non-fuzzing rules around private key validity). Other changes are required because, as a result of that, we now have sha256 only return hashes with one byte set to 1-0xff instead of 0-0xff as we did previously.